multiterm/src/Makefile.am

```diff
@@ -24,6 +24,8 @@ multiterm_la_SOURCES = \
 	terminal.vala
 
 multiterm_la_CFLAGS = \
+	-DGETTEXT_PACKAGE='"$(GETTEXT_PACKAGE)"' \
+	-DLOCALEDIR='"$(localedir)"' \
```

Comment:
Alternatively you could probably have a config.h VAPI module, something like this:
```vala
[CCode(cheader="config.h", lower_case_cprefix="")]
namespace Config {
  const string GETTEXT_PACKAGE;
  const string LOCALEDIR;
}
```